pdns/recursordist/docs/manpages/rec_control.1.rst

```diff
@@ -74,23 +74,47 @@ dump-cache *FILENAME*
     also dumped to the same file. The per-thread positive and negative cache
     dumps are separated with an appropriate comment.
 
+    **Warning**: :program:`pdns_recursor` often runs in a chroot. You can
```

Comment:
in Sphinx, there is a warning command, but I would suggest using note here like this:

```rst
.. note::
  
  :program:`pdns_recursor` often runs in a chroot.
  You can retrieve the file using::

    rec_control dump-cache /tmp/file
    mv /proc/$(pidof pdns_recursor)/root/tmp/file /tmp/filename
```

Reply:
Happy to do that. The reason i pulled in `**Warning**` is because I saw it used later:

https://github.com/powerdns/pdns/blame/f52b0223fe256de9436bfa71fb3ad968d3e83bde/pdns/recursordist/docs/manpages/rec_control.1.rst#L231

Also note: note:
https://github.com/powerdns/pdns/blame/f52b0223fe256de9436bfa71fb3ad968d3e83bde/pdns/recursordist/docs/manpages/rec_control.1.rst#L229

I'm happy to change those in a distinct PR if that's desirable (I'd do it roughly at the same time as updating this one...)